app/assets/appfilter.xml

```diff
@@ -5543,7 +5543,39 @@
   <item component="ComponentInfo{me.tylerbwong.stack/me.tylerbwong.stack.ui.MainActivity}" drawable="stack" name="Stack" />
   <item component="ComponentInfo{com.standardnotes/com.standardnotes.MainActivity}" drawable="standard_notes" name="Standard Notes" />
   <item component="ComponentInfo{com.vitotechnology.StarWalk2Free/com.google.firebase.MessagingUnityPlayerActivity}" drawable="star_walk_2" name="Star Walk 2" />
-  <item component="ComponentInfo{com.starbucks.cn/com.starbucks.cn.home.revamp.launch.RevampLaunchActivity}" drawable="starbucks" name="starbucks" />
+  <item component="ComponentInfo{com.starbucks.vn/com.starbucks.vn.MainActivity}" drawable="starbucks" name="Starbucks" />
```

Comment:
* Starbucks Hong Kong
* Starbucks Peru

```suggestion
  <item component="ComponentInfo{ccom.starbucks.pe/com.starbucks.emea.features.splash.SplashActivity}" drawable="starbucks" name="Starbucks" />
  <item component="ComponentInfo{com.starbucks.hk/com.starbucks.MainActivity}" drawable="starbucks" name="Starbucks" />
  <item component="ComponentInfo{com.starbucks.vn/com.starbucks.vn.MainActivity}" drawable="starbucks" name="Starbucks" />
```

Reply:
Most likely you need to change `ccom` to `com`.

If you still have all these apps installed, please make sure the icon is applied to all of them.